### PR TITLE
Reworked: Balena

### PR DESCRIPTION
**Pull request by rusko124**
These changes provide an opportunity to install the Screenly-OSE on Balena Cloud.

> The INITSYSTEM functionality has been completely removed (https://www.balena.io/docs/reference/base-images/base-images/)

Thus, we do not use the services anymore.

Edits:
- multi-container #1080 
- reworked `./bin/start-balena.sh`
- Hidden button `Reset the network` if it is Balena application.
balena

System info:
balena_system
The system information doesn't display the Screenly version and current logs(but all logs we see in Balena admin)

### CODE REVIEW

**Comment by vpetersson** [comment]
@rusko124 is this ready to be merged? Also, could you fix the Snyk warning?

**Comment by JeffResc** [comment]
Installed this on my devices as a fix for #1102, can conclude that this is working on all four of my production devices.

**Comment by JeffResc** [comment]
I take my earlier comment back. Upon restarting the device, it will issue the following to the log around once a second: `09.05.19 15:24:30 (-0400)  screenly-viewer  Still continue...;. I believe this is due to some files being stored to the `/tmp` directory which has some persistency issues as [described here](https://www.balena.io/docs/learn/develop/runtime/#inconsistency-in-tmp-directory).

**Comment by rusko124** [comment]
@vpetersson I'll ping you when the PR will ready to merge

**Comment by rusko124** [comment]
@vpetersson I think the PR ready to merge. 
I tested manually:
- All types of assets are displayed(for video expose gpu_mem = 64)
- Public and internal ips are work
- Settings apply and save the configuration
- System info displays almost all information(except logs and current version)
- After rebooting the device everything works (Fixed after @JeffResc comment above)

Any questions or suggestions?

**Comment by vpetersson** [comment]
@rusko124 perfect. I'll merge it and hopefully we get some feedback before the next release. @JeffResc pls confirm.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  screenly-server:
+    build:
+      context: .
+      dockerfile: Dockerfile.template
+    environment:
+      - HOME=/data
+      - LISTEN=0.0.0.0
+      - PYTHONPATH=/data/screenly
+      - SCREENLYSERVICE=server
+      - SWAGGER_HOST=ose.demo.screenlyapp.com
```
Comment (vpetersson) [inline docker-compose.yml]: This isn't right tho. We should serve this locally.

**Pull request merged**